### ISSUE TITLE
Question about CHANGELOG.md

### ISSUE THREAD

**Issue by cb372** [47degrees/.github#7]
This is looking great!

But I'm about confused about the use of release-drafter and updating the `CHANGELOG.md` file. If we're using release-drafter, why don't we use the GitHub project's Releases page to publish our changelog? In other words, use release-drafter the way it is designed. Is there any benefit to putting the changes in a file inside the repo?

Maybe I'm misunderstanding the workflow definitions, but it looks like we are generating both release drafts *and* the CHANGELOG file, with quite similar contents?

**Comment by juanpedromoreno** [comment]
That's a great question @cb372 

I'd say one of the potential use cases for maintaining both, it's is to use the `CHANGELOG.md` file as a section in the microsite, for example. It's not something critical, and to be honest, we could simplify it if it's not a useful feature.

**Comment by alejandrohdezma** [comment]
I'll say that it is useful to maintain both for old projects. Those that maybe haven't been using Github Releases or that do not have descriptions on them. For those, it can be useful to maintain both changelog and releases.

Nevertheless, it would be awesome if we could "blacklist" some of the workflows so we could just stop generating changelog for new repos.

**Comment by juanpedromoreno** [comment]
Yeah, the possibility to choose which workflows are applied to a repo would be awesome :)

**Comment by alejandrohdezma** [comment]
I'm working on a Github Action to simplify all these that would allow that (among other things) stay tuned!! :)
